#!/usr/bin/python3

import sys


def safe_function(fct, *args):
    """DGYFUDGH JDFLJH HIIR HDFJK.

    Args:
        fct: DHFYUEDU YR7Y GASODO EIZ.
        args: GDFYUGVRIB GBUVR .

    Returns:
        shfuhi huidhfuui hr uhe euiy uehfigcsdG AJFG RH.
    """
    try:
        result = fct(*args)
        return (result)
    except:
        print("Exception: {}".format(sys.exc_info()[1]), file=sys.stderr)
        return (None)

